docs/dataset/advanced-usage/extract-metadata-text.md

```diff
@@ -0,0 +1,267 @@
+---
+icon: kolena/comparison-16
+---
+
+# :kolena-comparison-20: Text Metadata Hydration
+
+This guide outlines how to configure the extraction of metadata from text data on Kolena. Follow the steps below
+to get started with Automatic Metadata Hydration for Text.
+
+## Configuring Metadata Extraction
+
+**Navigate to Dataset Details**: Scroll down to the "Details" page of your dataset.
+
+<figure markdown>
+![Navigating to Configuration](../../assets/images/navigate-to-text-extraction-config.gif)
+<figcaption>Navigating to Metadata Configuration</figcaption>
+</figure>
+
+**Select Text Fields and Properties**: Identify and select the text fields from your dataset that you want to analyze.
+Also select the properties of the fields you wish to extract.
+
+<figure markdown>
+![Select Properties of Text Fields](../../assets/images/select-text-extraction-properties.gif)
+<figcaption>Select Specific Properties of Interest For Relevant Fields</figcaption>
+</figure>
+
+**Edit Metadata Configuration**: To make additional metadata visible (or to hide existing metadata),
+the configuration can be edited. If it is desired to add properties without re-running the extraction process,
+the box that says "Run extractions on save" must be **unchecked**. This will add/remove metadata properties.
+
+<figure markdown>
+![Edit Which Properties Are Visible](../../assets/images/edit-text-extraction-properties.gif)
+<figcaption>Add/Remove Text Metadata by Reconfiguring Extraction</figcaption>
+</figure>
+
+!!! example
```

Comment:
Should we add a sentence or two here explaining what we are seeing here? 
I think we should at least highlight which fields here are extracted by Kolena and which fields are not.

Reply:
I think my comment was not too clear.  I meant to add a sentence in the example image.  It would be nice to point out that the purple fields are the ones Kolena has extracted.

Reply:
<img width="700" alt="image" src="https://github.com/kolenaIO/kolena/assets/17771952/fac06fc9-2b99-4a15-9b85-7e8f1f062e84">

Reply:
Ah I see - I've added to the caption here.